src/main/java/org/elearning/portfolio/Beans.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+    http://www.springframework.org/schema/beans/spring-beans-3.0.xsd ">
+
+    <!-- Initialization for data source -->
+    <bean id="dataSource"
+          class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+        <property name="driverClassName" value="com.mysql.jdbc.Driver"/>
+        <property name="url" value="jdbc:h2:file:./target/portfolio-db"/>
+        <property name="username" value="root"/>
```

Comment:
User name i hasło powinno być takie jak skonfigurowałeś podczas inicjalizacji bazy danych przez flyway'a w pom.xml. Username to "sa" Hasło to ""